### PR TITLE
PG16: Regression Tests

### PR DESCRIPTION
**Pull request by fabriziomello**
Added missing template output tests and also converted others to
template due to output changes.

Disable-check: force-changelog-file

### CODE REVIEW

File: tsl/test/expected/compression_errors-16.out [hunk 1]
```diff
@@ -0,0 +1,829 @@
...
+        1
+(2 rows)
+
+EXPLAIN (COSTS OFF, TIMING OFF, SUMMARY OFF) SELECT DISTINCT 1 FROM test;
+                 QUERY PLAN                 
+--------------------------------------------
+ Limit
+   ->  Result
+         ->  Seq Scan on _hyper_31_19_chunk
+(3 rows)
```
Comment (svenklemm) [inline tsl/test/expected/compression_errors-16.out:593]: This looks not correct

Comment (svenklemm) [reply tsl/test/expected/compression_errors-16.out:593]: nvm PG16 seems to have learned some new tricks with distinct queries

File: tsl/test/shared/expected/dist_fetcher_type-16.out
Comment (svenklemm) [inline tsl/test/shared/expected/dist_fetcher_type-16.out:1]: dist tests have been removed from regresscheck-shared so this file is obsolete now

**Review by svenklemm — APPROVED** [review]

File: tsl/test/shared/expected/dist_fetcher_type-16.out
Comment (fabriziomello) [reply tsl/test/shared/expected/dist_fetcher_type-16.out:1]: Removed!

File: tsl/test/expected/plan_skip_scan-16.out [hunk 2]
```diff
@@ -0,0 +1,3979 @@
...
+               ->  Index Only Scan using _hyper_1_3_chunk_skip_scan_ht_dev_name_idx on _hyper_1_3_chunk (actual rows=1 loops=1)
+                     Index Cond: ((dev_name > NULL::text) AND (dev_name IS NULL))
+                     Heap Fetches: 1
+         ->  Custom Scan (SkipScan) on _hyper_1_4_chunk (actual rows=1 loops=1)
+               ->  Index Only Scan using _hyper_1_4_chunk_skip_scan_ht_dev_name_idx on _hyper_1_4_chunk (actual rows=1 loops=1)
+                     Index Cond: ((dev_name > NULL::text) AND (dev_name IS NULL))
+                     Heap Fetches: 1
+(19 rows)
+
+-- test constants in ORDER BY
```
Comment (jnidzwetzki) [inline tsl/test/expected/plan_skip_scan-16.out:3248]: For some reason, PG 16 does not use the SkipScan node for the following query. Instead, an Index Scan is used. Might be worth checking if this is just a cost-based decision or if the path is not generated.

**Review by jnidzwetzki — APPROVED** [review]
One comment, rest looks good to me

File: tsl/test/expected/plan_skip_scan-16.out (hunk 2 above)
Comment (fabriziomello) [reply tsl/test/expected/plan_skip_scan-16.out:3248]: Good catch ... will merge this PR and investigate it

**Pull request merged**